### ISSUE TITLE
[Device Support Request] Unsure about right way to define triggers for scene control buttons

### ISSUE THREAD

**Issue by Emrvb** [zigpy/zha-device-handlers#1040]
I'm working on the device handler for several iCasa remotes and I'm not really happy about how the scene buttons are exposed now. https://github.com/zigpy/zha-device-handlers/pull/1039

I've currently defined short press and long press triggers for the scene buttons but that doesnt really reflect what they do: "recall" and "store". In my opinion this could create confusion for users when integrating these devices with home assistant.

In short: are there proper triggers for scene buttons?

**Comment by MattWestb** [comment]
I think the "real" scene switches (not tuya ones that is sending on/off/toggle on multiple end points) is sending add and recalling to its bonded group but the question is it possible setting (bonding one group) to the scene cluster and then doing the saving and recalling the scenes from the remote ?
Its shall being possible doing the same with ZHA but all commands is not implanted to doing that for the moment.

If like using the Scene switch as one HA switch (not light commands) its only triggering the automatons on the commands received from the switch is not braking the light commands its sending to its bonded group.

For the naming i think the normal is numbers for HA multi switches (tuya and Xiaomi) but in the end its the devs that have the last world.

**Comment by Emrvb** [comment]
These particular remotes create a group to issue commands to. They also respond properly to cluster commands, so you can add and remove devices from the home assistant ui. I was really happy finding that out after I had to reset and pair a hard to reach device.

In my opinion, the way these devices work, independent from the coordinator, can be a bit confusing to people new to the zigbee ecosystem. I think the triggers/events in home assistant should be descriptive of what these buttons actually (can) do.

**Comment by MattWestb** [comment]
Not what can, what is doing !!

Some user like one one button "SOS button" sending 1 to 6 clicks for doing automation for triggering all crazy things and is not working if node red is having problems but its up to the user.

And its not easy finding good working light switches that is doing real bindings to groups that working well and not costing like one Lamborghini.

Back to your problem with naming. 
If using the switch to controlling lights its shall not using the scenes commands for automation = no need implanting it.
But if have more switches and like have one controlling lights but not using scenes  (or only on/off and dim) its good having the extra scene buttons configurated with automation trigger so i think its good implanting them and not using  them if not like but then user can so if they want to do it

Then back to the naming: I think Scene X add and recalling is better names then you is knowing the function and only numbers is not telling the "real" functions f them if you using its real function

**Comment by Emrvb** [comment]
Thats exactly my point. Well, calling it button 1, 2, 3, etc. instead of Scene X doesn't even matter to me that much. But calling the "store" function "long press" might cause some serious initial confusion for people that use these and upgrade their setup by adding home assistant to the mix.

I believe deconz uses store and recall as well, although I don't know if that terminology is exposed to home assistant.
